webdriver/tests/bidi/network/conftest.py

```diff
@@ -15,7 +15,7 @@ def fetch(bidi_session, top_context):
     """Perform a fetch from the page of the provided context, default to the
     top context.
     """
-    async def fetch(url, method="GET", headers=None, context=top_context):
+    async def fetch(url, method="GET", headers=None, context=top_context, timeout=1000):
```

Comment:
Lets make sure that we use the [timeout_multiplier](https://github.com/web-platform-tests/wpt/blob/d888ebb5b876d58a23934a10e7757421ab40e7c3/webdriver/tests/support/fixtures.py#L143) here as well. Otherwise this will break tests for slower running builds.

Reply:
good point! Done

Reply:
Not sure if only the default value should be multiplied or all of them.

Reply:
It should not just be the default but all values. So the patch looks right.